listings_app/views.py

```diff
@@ -0,0 +1,5 @@
+from django.http import HttpResponse
+
+
+def homepage(request):
+    return HttpResponse("<h1>Hello! I am the homepage</h1>")
```

Comment:
Same as above.

This is a minor thing, but it is best to be consistent with formatting. If a file uses single quotes, you should try to use single quotes elsewhere in the file. You have mixed single and double quotes. See PEP8's recommendation here: https://www.python.org/dev/peps/pep-0008/#string-quotes